src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -71,27 +136,135 @@ const ChatItemTargetText = ({
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
   avatar
 }: ChatItemMyTextProps) => {
+  const navigation = useNavigation();
+
   const renderAvatar = React.useCallback(() => {
     if (isContinuous) return <View style={styles.avatar} />;
     return <View style={styles.mr8}>{avatar}</View>;
   }, []);
 
+  const handleTextContainerStyle = () => {
+    const isAttachment = attachments.length > 0;
+    const paddingStyle = {
+      paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+      paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+    };
+    return [styles.textContainer, paddingStyle];
+  };
+
+  const onOpenMediaPreview = (medias, index, navigationLocal) => {
+    if (medias.find((media) => media.type === 'video')) {
+      navigationLocal.push('VideoViewer', {
+        title: 'Video',
+        url: medias[index].video_path
+      });
+    } else {
+      navigationLocal.push('ImageViewer', {
+        title: 'Photo',
+        index,
+        images: medias
+          .filter((media) => media.type === 'image')
+          .map((media) => ({url: media.asset_url}))
+      });
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This function handles navigation to media preview based on the type of media. It's good to see that you're considering different types of media. However, it might be better to split this function into two separate functions, one for handling video navigation and another for image navigation. This would make the code easier to read and maintain.

```diff
-  const onOpenMediaPreview = (medias, index, navigationLocal) => {
-    if (medias.find((media) => media.type === 'video')) {
-      navigationLocal.push('VideoViewer', {
-        title: 'Video',
-        url: medias[index].video_path
-      });
-    } else {
-      navigationLocal.push('ImageViewer', {
-        title: 'Photo',
-        index,
-        images: medias
-          .filter((media) => media.type === 'image')
-          .map((media) => ({url: media.asset_url}))
-      });
-    }
-  };
+  const onOpenVideoPreview = (medias, index, navigationLocal) => {
+    navigationLocal.push('VideoViewer', {
+      title: 'Video',
+      url: medias[index].video_path
+    });
+  };
+
+  const onOpenImagePreview = (medias, index, navigationLocal) => {
+    navigationLocal.push('ImageViewer', {
+      title: 'Photo',
+      index,
+      images: medias
+        .filter((media) => media.type === 'image')
+        .map((media) => ({url: media.asset_url}))
+    });
+  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->